docs/guides/getting-started-example.md

```diff
@@ -0,0 +1,239 @@
+
+## View the process instance and start job workers
+
+A process instance is now running in the engine.
+
+You can view the process instance in **Operate**, the visual operations tool, by navigating to:  
+[http://localhost:8080/operate](http://localhost:8080/operate)  
+Login with the credentials: `demo` / `demo`.
+
+There you will see an active process instance. (Note: when the workers are running, process instances will be completed immediately and further process instances will not appear as active).
+
+Next, start the job workers to allow them to perform the work for the service tasks. The workers are configured to connect to the locally-running engine and retrieve available work for the process instance.
+
+<Tabs groupId="language" defaultValue="javaspring" values={
+[
+{ label: 'Java + Spring', value: 'javaspring', },
+{ label: 'Node.js', value: 'nodejs', },
+] }>
+<TabItem value="javaspring">
+
+<ol>
+<li>
+Change into the Spring SDK directory:
+
+```bash
+cd java
+```
+
+      </li>
+      <li>
+        Start the workers with the command:
+
+```bash
+mvn spring-boot:run
+```
+
+      </li>
+    </ol>
+
+  </TabItem>
+
+  <TabItem value="nodejs">
+    <ol>
+      <li>
+        Change into the Node.js SDK directory:
+```bash
+cd nodejs
+```
+      </li>
+      <li>
+        Install dependencies with the command:
+```bash
+npm i
+```
+      </li>
+      <li>
+        Start the workers with the command:
+```bash
+npm start
+```
+      </li>
+    </ol>
+  </TabItem>
+   </Tabs>
+
+The workers start, connect to the engine, and request work. You will see the workers processing the jobs for the process instance.
+
+<Tabs groupId="language" defaultValue="javaspring" values={
+[
+{ label: 'Java + Spring', value: 'javaspring', },
+{ label: 'Node.js', value: 'nodejs', },
+] }>
+<TabItem value="javaspring">
+
+    ![Worker output](./img/get-started-example-output-java.png)
+
+   </TabItem>
+   <TabItem value="nodejs">
+
+    ![Worker output](./img/getting-started-example-output.png)
+
+   </TabItem>
+</Tabs>
+
+## Complete!
+
+Your active process instance in Operate will now transition to completed. You have completed your first process instance by:
+
+1. Running Camunda 8 Run locally.
+2. Deploying a process model.
+3. Starting a process instance.
+4. Completing the service tasks in the process instance using job workers.
+
+## Next steps
+
+To create a new Spring project, refer to [our guide using Spring](/guides/getting-started-java-spring.md).
+
+For further information about the Node.js SDK, refer to [the Camunda 8 JavaScript SDK API documentation](https://camunda.github.io/camunda-8-js-sdk/).
+
+## Teardown
+
+You can stop Camunda 8 Run by executing:
+
+    <Tabs groupId="os" defaultValue="maclinux" values={
+
+[
+{ label: 'Mac OS + Linux', value: 'maclinux', },
+{ label: 'Windows', value: 'windows', },
+] }>
+<TabItem value="maclinux">
+```bash
+./c8run/c8run stop
```

Comment:
🔧 c8run depends on the current dir, it fails when run like this, thus we need to change the dir 😬 
```suggestion
cd c8run && ./c8run stop
```